metadata.rb

```diff
@@ -4,7 +4,7 @@
 maintainer_email 'mklishin@pivotal.io'
 license 'Apache-2.0'
 description 'Installs and configures RabbitMQ server'
-version '5.8.0'
+version '5.8.1'
```

Comment:
`5.8.0` hasn't shipped yet. Please undo this and never mess with versions in pull requests. It's up to the maintainer to decide what the version should be and why. Contributors simply don't have the context to make this call.

Reply:
apologies, force of habbit from internal development workflows... reverted

Reply:
Hm, I still see a single commit in this PR. Did you mean to force push the branch? I wouldn't mind reverting this myself, by the way.